shared/ui-components/src/navigations/LanguageNav/LanguageNav.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+
+import { LanguageNav } from "./LanguageNav"
+
+export default {
+  title: "Prototypes/LanguageNav",
+  decorators: [(storyFn: any) => <div style={{ padding: "1rem" }}>{storyFn()}</div>],
+}
+
+export const Default = () => (
+  <LanguageNav>
+    <li>
+      <a href="">English</a>
+    </li>
+    <li>
+      <a href="">Spanish</a>
+    </li>
+    <li>
+      <a href="">Chinese</a>
+    </li>
+    <li>
+      <a href="">Vietnamese</a>
+    </li>
```

Comment:
This story should show how to use the actual UI element that it renders.

For example, the code here should be:

```jsx
const items = [...];
<LanguageNav item={items} />
```

That way a developer looking through story book can see both how the element looks and how to use it. In your example here, all the `<li>` elements are unused (since `LanguageNav` never renders `children`), and we don't see items actually getting rendered.